lib/screens/leaderboard.dart

```diff
@@ -140,3 +174,4 @@ createPlayerCard(String team, String rank) {
     ),
   );
 }
+*/
```

Comment:
Not sure why it says there's no newline at the end of the file here, but does `createPlayerCard()` need to be a comment?